scripts/us_fema/national_risk_index/README.md

```diff
@@ -0,0 +1,66 @@
+# Importing FEMA National Risk Index (NRI) Data
+
+This directory imports the [FEMA NRI](https://hazards.fema.gov/nri/) dataset
+into Data Commons. The study includes relative measures of risks from 18 hazards
+to the US at the county and tract level, as well as data on individual hazards
+and their estimated risk to those affected.
+
+Source data can be downloaded from this [download page](
+https://hazards.fema.gov/nri/Content/StaticDocuments/DataDownload//NRI_Table_CensusTracts/NRI_Table_CensusTracts.zip
+).
+
+# Instructions
+
+## Generating Artifacts
+
+To generate the schema and the TMCF for the import, run
+```
+python3 generate_schema_and_tmcf.py
+```
+
+To clean the source data, run
+```
+python3 process_data.py
+```
+
+Running these 2 commands will generate the following files under the `output`
+directory
+
+1. `fema_nri_schema.mcf` holds the Schema for the StatisticalVariables in the
+dataset
+1. `fema_nri_counties.tmcf` holds the TMCF nodes for importing the dataset at
+the county level (a.k.a. `NRI_Table_Counties.csv`)
```

Comment:
nit: nri_table_counties ?

Reply:
data has the capitalization on source. should I add a step to rename it to lowercase?

Reply:
more generally, here, I was referring to the source data, rather than the generated. I will update capitalization since it is clearer. thanks for the catch!